scripts/apps/search/directives/RawSearch.js

```diff
@@ -1,12 +1,22 @@
-RawSearch.$inject = ['asset', '$location'];
+/**
+ * @memberof superdesk.apps.search
```

Comment:
It's ok, what you chose makes more sense, but it won't work unfortunately. I've made the same mistake. I am hoping to have a guide and the documentation website up by end of sprint.